src/main/java/frc/robot/Constants.java

```diff
@@ -49,25 +53,33 @@
 
         public static final double STOP_CLIMBER = 149.5; // [s]
 
-        public static class Vision {
-            public static final double CAMERA_HEIGHT = 0.767; // [m]
-            public static final double TARGET_HEIGHT_FROM_GROUND = 2.64; // [m]
-            public static final double CAMERA_PITCH = 30; // Pitch of the vision. [deg]
-            public static final double DIAG_FOV = 75; // Diagonal FOV. [deg]
-            public static final double LED_RANGE = 6; // Visible range of LEDs. [m]
-            public static final int CAM_RESOLUTION_WIDTH = 640; // Width of camera resolution. [pixel]
-            public static final int CAM_RESOLUTION_HEIGHT = 480; // Height of camera resolution. [pixel]
-            public static final double MIN_TARGET_AREA = 10; // Minimal area of target. [pixel^2]
-            public static final double TARGET_WIDTH = 1.36; // Width of vision target strip. [m]
-            public static final double TARGET_HEIGHT = 0.05; // Height of the vision target strip. [m]
-
-            public static final Pose2d HUB_POSE = new Pose2d( // Position of the hub relative to the field.
-                    new Translation2d(FIELD_LENGTH / 2, FIELD_WIDTH / 2), new Rotation2d());
-            public static final Transform2d CAMERA_TO_ROBOT = new Transform2d(
-                    new Translation2d(0.038, 0.171), new Rotation2d()); // Position of the vision relative to the robot.
-
-            public static final SimVisionTarget SIM_TARGET_HUB = new SimVisionTarget( // Hub target for vision simulation.
-                    HUB_POSE, TARGET_HEIGHT_FROM_GROUND, TARGET_WIDTH, TARGET_HEIGHT);
-        }
+        public static final double SIMULATION_LOOP_PERIOD = 0.02; // [s]
+
+        public static final int ZERO_POSITION = 0;
+
+        public static final int ZERO_POSITION_TOLERANCE = 0;
```

Comment:
You'll probably never be _exactly_ on point. Tolerance should be greater than zero (although not by a lot).